pallets/cash/src/lib.rs

```diff
@@ -470,6 +473,29 @@ impl<T: Config> frame_support::traits::EstimateNextSessionRotation<T::BlockNumbe
     }
 }
 
+// weight is 0 if request is invalid
```

Comment:
Is 0 the right default? Maybe make it semi-high just in case this comes up in the real world for some reason?